lib.php

```diff
@@ -653,16 +653,11 @@ function block_coursefeedback_get_editerrors($feedbackid) {
 function block_coursefeedback_set_active($feedbackid) {
     global $DB;
     if ($feedbackid == 0 || $DB->record_exists("block_coursefeedback", array("id" => $feedbackid))) {
```

Comment:
Wenn du schon dabei bist, könnte man die Bedingung noch invertieren, damit einen frühen `return false` machen und so eine Ebene der Verschachtelung sparen:
```php
if ($feedbackid != 0 && !$DB->record_exists("block_coursefeedback", array("id" => $feedbackid))) {
    return false;
}
$oldfeedbackid = get_config("block_coursefeedback", "active_feedback");
// ...
return true;
```

Reply:
Das ist ein Programmierkonzept zur besseren Lesbarkeit. Programmatisch sind die beiden Versionen gleich, aber der Abbruch in deiner Variante kann von anderen Programmierer:innen übersehen werden. Solange die Notwendigkeit nicht besteht würde ich das nicht anfassen.

Reply:
Ich kann beides nachvollziehen. Ich habe den einfachen Weg gewählt und nichts geändert :)